src/kernel/bitcoinkernel.h

```diff
@@ -933,10 +944,10 @@ BITCOINKERNEL_API const btck_BlockHash* BITCOINKERNEL_WARN_UNUSED_RESULT btck_bl
  * @brief Create options for the chainstate manager.
  *
  * @param[in] context          Non-null, the created options and through it the chainstate manager will
-                               associate with this kernel context for the duration of their lifetimes.
- * @param[in] data_directory   Non-null, path string of the directory containing the chainstate data.
```

Comment:
I don't think this change is right. The path can not be null or empty, because that will fail later on with:

```
unknown location(0): fatal error: in "fs_tests/fsbridge_stem": std::filesystem::__cxx11::filesystem_error: filesystem error: cannot make absolute path: Invalid argument []

Reply:
> How did you get that fatal error?

It is hidden in the error message, but it says "cannot make absolute path".

Did you actually call the full kernel function here, including:

```cpp
        fs::path abs_data_dir{fs::absolute(fs::PathFromString({data_dir, data_dir_len}))};
        fs::create_directories(abs_data_dir);
```

In any case, even if it doesn't crash, I wonder what the valid meaning is here, of passing an empty string?